RNWCPP/ReactUWP/Executors/WebSocketJSExecutorUwp.cpp

```diff
@@ -213,12 +213,17 @@ winrt::Windows::Foundation::IAsyncAction WebSocketJSExecutor::ConnectAsync(const
 
   SetState(State::Connected);
 
-  PrepareJavaScriptRuntime();
-
-  SetState(State::Running);
+  if (PrepareJavaScriptRuntime())
+  {
+    SetState(State::Running);
+  }
+  else
+  {
+    OnHitError("Create a new JS runtime timeout and Executor instance is not connected to a WebSocket endpoint.");
+  }
 }
 
-void WebSocketJSExecutor::PrepareJavaScriptRuntime()
+bool WebSocketJSExecutor::PrepareJavaScriptRuntime()
 {
   auto timeout = std::chrono::milliseconds(250);
```

Comment:
Should we bump to the retry count a little bit too?  I haven't seen this error so maybe my machine is fast enough for retries.  I do get errors when chrome wasn't running at all and it booted for debugging - but it launches to an error state that the JS will never be able to connect to

Reply:
3*250ms is enough of non debugging setup. I don't know if I need to increase the retry count to support debugging.

Reply:
I increase retry count and timeout would be from 750ms to 5s